### PR TITLE
Fixes minor typos in spell descriptions

### PR DESCRIPTION
**Pull request by asparkman**
Wanted to say this tool has been a huge help in getting to spell descriptions in the middle of sessions!

I ran the _posts files through a spell checker, and this is what I got.

### CODE REVIEW

File: _posts/2015-01-07-crown-of-madness.markdown [hunk 1]
```diff
@@ -16,4 +16,4 @@ tags: [bard, paladin (oathbreaker), sorcerer, warlock, wizard, level2, enchantme
...
+One humanoid of your choice that you can see within range must succeed on a Wisdom saving throw or become charmed by you for the duration. While the target is charmed in this way, a twisted crown of jagged iron appears on its head, and a madness glows in its eyes. The charmed target must use its action before moving on each of its turns to make a melee attack against a creature other than itself that you mentally choose. The target can act normally on its turn if you choose no creature or if none are within its reach. On your subsequent turns, you must use your action to maintain control over the target, or the spell ends. Also, the target can make a Wisdom saving throw at the end of each of its turns. On a success, the spell ends.
```
Comment (asparkman) [inline _posts/2015-01-07-crown-of-madness.markdown:19]: non -> none

File: _posts/2015-07-30-bigbys-hand.markdown [hunk 2]
```diff
@@ -28,6 +28,6 @@ When you cast the spell and as a bonus action on your subsequent turns, you can
...
 
-**Interposing Hand.** The hand interposes itself between you and a creature you choose until you give the hand a different  command. The hand moves to stay between you and the target, providing you with half cover against the target. The target can't move through the hand's space if its Strength score is less than or equal to the hand's Strength score. If its Strength score is higher than the hand's Strength score, the target can move toward you through the hand's space, but that space is difficult terrain for the target.
+**Interposing Hand.** The hand interposes itself between you and a creature you choose until you give the hand a different command. The hand moves to stay between you and the target, providing you with half cover against the target. The target can't move through the hand's space if its Strength score is less than or equal to the hand's Strength score. If its Strength score is higher than the hand's Strength score, the target can move toward you through the hand's space, but that space is difficult terrain for the target.
```
Comment (Traneptora) [inline _posts/2015-07-30-bigbys-hand.markdown:31]: What changed here?

Comment (asparkman) [reply _posts/2015-07-30-bigbys-hand.markdown:31]: "different  command" -> "different command"  double space to single.

**Review by Traneptora — APPROVED** [review]
Thank you! With a work like this with lots of things by lots of people, there will be typos, so this sort of thing is helpful. I appreciate it :)

**Pull request merged**